src/NServiceBus.Core/Persistence/Development/SagaPersister/DevelopmentSagaPersistence.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace NServiceBus.Features
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Runtime.Serialization.Json;
+    using NServiceBus.Sagas;
+
+    class DevelopmentSagaPersistence : Feature
+    {
+        internal DevelopmentSagaPersistence()
+        {
+            DependsOn<Sagas>();
+            Defaults(s => s.Set<ISagaIdGenerator>(new DevelopmentSagaIdGenerator()));
+        }
+
+        /// <summary>
+        /// See <see cref="Feature.Setup" />.
+        /// </summary>
+        protected internal override void Setup(FeatureConfigurationContext context)
+        {
+            var rootDir = @"c:\dev\storage";
```

Comment:
This should probably be configurable should it? This might work for many devs but I can see this being a problem in a corporate dev environment.